src/rngs/std.rs

```diff
@@ -10,6 +10,7 @@
 
 use crate::{CryptoRng, Error, RngCore, SeedableRng};
 
+#[cfg(feature = "getrandom")]
 pub(crate) use rand_chacha::ChaCha12Core as Core;
```

Comment:
This looks wrong: `StdRng` does not depend on `getrandom`.

Reply:
This alias is used here: https://github.com/SUPERCILEX/rand/blob/e0c8388fc8dfbb1c9206757112ac86f0e2c690f3/src/rngs/thread.rs#L70

Which uses `OsRng` and so requires `getrandom`. Kinda weirdly organized, I agree.